### PR TITLE
Add reference to Gitlab on About/contributor-guides/for-staff

### PR DESCRIPTION
**Pull request by ElenaFdR**
-----
[View rendered contributor-guides/for-staff.md](https://github.com/publiccodenet/about/blob/gitlab/contributor-guides/for-staff.md)

### CODE REVIEW

**Comment by bvhme** [comment]
Are there any codebase stewardship related things we're doing on GitLab?

File: contributor-guides/for-staff.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Here are some important things to figure out in the first couple of days:
 * Attend morning standup ritual that is in the team calendar
 * Attend sprint retrospective & planning - review the kanbans beforehand
 * Tentoo NMBRS ESS (employee self service) mobile app - to [request leave](../activities/staff-management/leave.md), [file expenses](../activities/staff-management/expense.md) and see your payslips
+* if you already have a Gitlab account, [join the Foundation for Public Code group](https://gitlab.com/publiccodenet) - we only use this for our codebase stewardship work, so it may not be relevant to your role
```
Comment (ElenaFdR) [inline contributor-guides/for-staff.md:22]:
```suggestion
* if you already have a Gitlab account, [join the Foundation for Public Code group](https://gitlab.com/publiccodenet) (note: we primarily work on Github)
```

Comment (bvhme) [reply contributor-guides/for-staff.md:22]: I'd suggest something stronger. As we do no work on GitLab.

```suggestion
* if you already have a Gitlab account you can [join the Foundation for Public Code group](https://gitlab.com/publiccodenet) although we don't do any work on GitLab.
```

File: contributor-guides/for-staff.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ Here are some important things to figure out in the first couple of days:
 * Attend morning standup ritual that is in the team calendar
 * Attend sprint retrospective & planning - review the kanbans beforehand
 * Tentoo NMBRS ESS (employee self service) mobile app - to [request leave](../activities/staff-management/leave.md), [file expenses](../activities/staff-management/expense.md) and see your payslips
+* if you already have a Gitlab account you can [join the Foundation for Public Code group](https://gitlab.com/publiccodenet) although we don't do any work on GitLab.
```
Comment (bvhme) [inline contributor-guides/for-staff.md:22]: Inconsistent brand name :)

```suggestion
* if you already have a GitLab account you can [join the Foundation for Public Code group](https://gitlab.com/publiccodenet) although we don't do any work on GitLab.
```

**Pull request merged**